src/cider/include/cider/processor/BatchProcessorContext.h

```diff
@@ -18,23 +18,60 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+#pragma once
 
 #ifndef CIDER_BATCH_PROCESSOR_CONTEXT_H
 #define CIDER_BATCH_PROCESSOR_CONTEXT_H
 
+#include <any>
 #include <functional>
 #include <memory>
 #include <optional>
 #include <vector>
 #include "cider/CiderAllocator.h"
-
+#include "exec/nextgen/context/Batch.h"
+#include "exec/operator/join/HashTableFactory.h"
 namespace cider::exec::processor {
+using CiderJoinBaseHashTableTemplate = cider_hashtable::BaseHashTable<
```

Comment:
why use 'Template' suffix?

Reply:
I am trying to make shortcut for this template class to avoid write long arguments in ```<>```.